### PR TITLE
[JSORC]: DB Regen

### LINKED ISSUE #833: Bring DB under JSORC's automation
Similar to how JSORC is automatically keeping redis alive right now.

### PR DESCRIPTION
**Pull request by amadolid**
Closes #833

### CODE REVIEW

File: jaseci_core/jaseci/svc/common.py [hunk 1]
```diff
@@ -9,11 +9,13 @@
 from jaseci.actions.live_actions import load_action_config
 from jaseci.svc.actions_optimizer.actions_optimizer import ActionsOptimizer
 from .state import ServiceState as Ss
-from .config import META_CONFIG, KUBERNETES_CONFIG
+from .config import META_CONFIG, KUBERNETES_CONFIG, DATABASE as db, POSTGRES_MANIFEST
```
Comment (ypkang) [inline jaseci_core/jaseci/svc/common.py]: Any particular reason you are renaming DATABASE as db here? It is not a big deal but i feel like it doesn't help with readability.

File: jaseci_core/jaseci/svc/common.py [hunk 2]
```diff
@@ -288,19 +290,53 @@ def __init__(self, meta):
...
 
     def build(self):
         try:
-            hook = self.build_context("hook")
-            config = hook.service_glob("META_CONFIG", META_CONFIG)
+            config = META_CONFIG
+            if self.has_db:
+                hook = self.build_context("hook")
+                config = hook.service_glob("META_CONFIG", META_CONFIG)
+                self.prometheus = self.meta.get_service("promon", hook)
```
Comment (ypkang) [inline jaseci_core/jaseci/svc/common.py:339]: Why is prometheus hoisted up here?

File: jaseci_core/jaseci/svc/config.py [hunk 3]
```diff
@@ -1,9 +1,138 @@
...
+USER = os.environ.get("POSTGRES_USER", "postgres")
+
+DATABASE = {
+    "enabled": os.environ.get("JSORC_DB_REGEN") == "true",
+    "pod": os.environ.get("JSORC_POD_NAME", "jaseci"),
+    "host": os.environ.get("POSTGRES_HOST", "jaseci-db"),
+    "db": os.environ.get("POSTGRES_DB", "postgres"),
+    "name": USER,
+    "user": USER,
+    "password": os.environ.get("POSTGRES_PASSWORD", "lifelogifyjaseci"),
```
Comment (ypkang) [inline jaseci_core/jaseci/svc/config.py]: Let's not have the username/pw as plain text here. Let's just not have default value.

Comment (ypkang) [reply jaseci_core/jaseci/svc/config.py]: Same with the USER field above.

File: scripts/jsorc/jaseci.yaml [hunk 4]
```diff
@@ -151,12 +151,24 @@ spec:
             - name: POSTGRES_HOST
               value: jaseci-db
 
+            - name: POSTGRES_DB
+              value: postgres
```
Comment (ypkang) [inline scripts/jsorc/jaseci.yaml:155]: Let's call this DBNAME.

**Review by ypkang — CHANGES_REQUESTED** [review]
* I left some comments inline.
* I tested this and it works fine.
* The main thing I'd like to change is the first time jaseci comes up, there are many errors in the logs. I think they are all related to can't access DB which is expected but it looks like something is going wrong when in reality, everything is working as expected. Can we suppress those errors from the logs?

File: jaseci_core/jaseci/svc/common.py (hunk 2 above)
Comment (amadolid) [reply jaseci_core/jaseci/svc/common.py:339]: Prometheus is part of JsOrc and pointed to JsOrc's Kubernetes so I just initialized it when JsOrc is automated

**Review by ypkang — CHANGES_REQUESTED** [review]
@amadolid did you see this comment?

The main thing I'd like to change is the first time jaseci comes up, there are many errors in the logs. I think they are all related to can't access DB which is expected but it looks like something is going wrong when in reality, everything is working as expected. Can we suppress those errors from the logs?

**Comment by amadolid** [comment]
> @amadolid did you see this comment?
> 
> The main thing I'd like to change is the first time jaseci comes up, there are many errors in the logs. I think they are all related to can't access DB which is expected but it looks like something is going wrong when in reality, everything is working as expected. Can we suppress those errors from the logs?

ow sorry, I've overlooked this part.
Will post update for this

**Comment by amadolid** [comment]
I have updated some approach so lessen the exceptions. On django related logs, I'm not sure yet if can suppress that without overriding django implementation. I think logging for db connection issue would be needed too as it would be the tracker on current jaseci build up

**Review by ypkang — APPROVED** [review]

**Pull request merged**